go.mod

```diff
@@ -0,0 +1,29 @@
+module github.com/revel/cmd
+
+go 1.13
```

Comment:
Shouldn't this be 1.12 if we try to build using 1.12?

Reply:
Fixed